tools/SetupFlow/DevHome.SetupFlow/ViewModels/SetupFlowViewModel.cs

```diff
@@ -29,9 +29,10 @@ public partial class SetupFlowViewModel : ObservableObject
     private readonly MainPageViewModel _mainPageViewModel;
     private readonly PackageProvider _packageProvider;
 
-    private readonly string _creationFlowNavigationParameter = "StartCreationFlow";
     private readonly string _configurationFlowNavigationParameter = "StartConfigurationFlow";
-    private readonly string _quickstartNavigationParameter = "StartQuickstartPlayground";
+    private readonly string _creationFlowNavigationParameter = "StartCreationFlow";
+
+    private readonly Dictionary<string, Action<string>> _navigationTargets = new();

```

Comment:
you can actually add the targets in during the creation of this. Something like this: https://github.com/microsoft/devhome/blob/f96f181c5c778d81e8d1e56edccd4f527717e7f5/common/Services/ComputeSystemService.cs#L28  but with a Dictionary.

Reply:
I tried this already.

    private readonly Dictionary<string, Action<string>> _myTargets = new()
    {
        { "StartQuickstartPlayground", StartQuickTestFlow},
    };

`{ "StartQuickstartPlayground", StartQuickTestFlow},` gives the error 

     The best overloaded method match has wrong signature for the initializer element. The initializable Add must be an accessible instance method.

I'm going with this.